plugins/grpc/python/v1.66.2/buf.plugin.yaml

```
@@ -0,0 +1,19 @@
+version: v1
+name: buf.build/grpc/python
+plugin_version: v1.66.2
+source_url: https://github.com/grpc/grpc
+description: Generates Python client and server stubs for the gRPC framework.
+deps:
+  - plugin: buf.build/protocolbuffers/python:v27.4
+output_languages:
+  - python
+spdx_license_id: Apache-2.0
+license_url: https://github.com/grpc/grpc/blob/v1.66.2/LICENSE
+registry:
+  python:
+    package_type: "runtime"
+    # https://github.com/grpc/grpc/tree/v1.66.2/src/python/grpcio#supported-python-versions
```

Looks like this anchor is gone - I think as a result of https://github.com/grpc/grpc/pull/37802.

```suggestion
    # https://github.com/grpc/grpc/blob/v1.66.2/src/python/grpcio/python_version.py#L19
```